packages/channels/botpress-channel-web/src/api.js

```diff
@@ -231,17 +230,14 @@ module.exports = async (bp, config) => {
     // We remove the password from the persisted messages for security reasons
     if (payload.type === 'login_prompt') {
       persistedPayload.data = _.omit(persistedPayload.data, ['password'])
-    }
-
-    if (payload.type === 'form') {
+    } else if (payload.type === 'form') {
       persistedPayload.data.formId = payload.formId
     }
 
-    const message = await appendUserMessage(userId, conversationId, persistedPayload)
-
-    Object.assign(message, {
-      __room: 'visitor:' + userId // This is used to send to the relevant user's socket
-    })
+    const message = {
+      ...(await appendUserMessage(userId, conversationId, persistedPayload)),
```

Comment:
it'll be better to remove `...(await appendUserMessage(` to variable for explain what is returned
```javascript
const understandableName = await appendUserMessage(userId, conversationId, persistedPayload)
const message = {
  ...understandableName,
  __room: `visitor:${userId}`
}
```